source/system-administrators/activities/language-support/add-new-language.rst

```diff
@@ -0,0 +1,152 @@
+:is-up-to-date: True
+
+.. index:: Adding a New Language, Language Support
+
+.. _language-support-add-new:
+
+=====================
+Adding a New Language
+=====================
+
+Crafter Studio has been translated into a number of languages.  Here's a list of currently supported languages along with the language codes used in Studio:
+
+* ``en`` - English
+* ``sp`` - Spanish
+* ``de`` - German
+* ``kr`` - Korean
+
+To reach more users, additional languages may need to be added in Crafter Studio.
+
+**Here are the steps to add a new language in Crafter Studio:**
+
+#. Add translations file for the Studio user dashboard ``studio-ui/static-assets/scripts/resources/locale-**.json``
+#. Add the translations file for everything else not in the dashboard ``studio-ui/static-assets/components/cstudio-common/resources/**/base.js``
+#. Add the new language to the ``get-available-languages`` API
+#. Add the new language to the react translations manager
+#. Update templates to add the new language
```

Comment:
Update templates to add the new language imports into the runtime (i.e. via `script[src]` elements)